homeassistant/components/openhome/media_player.py

```diff
@@ -94,13 +96,14 @@ def update(self):
         self._source_names = source_names
 
         if self._source["type"] == "Radio":
-            self._supported_features |= SUPPORT_STOP | SUPPORT_PLAY
-        if self._source["type"] in ("Playlist", "Cloud"):
+            self._supported_features |= SUPPORT_STOP | SUPPORT_PLAY | SUPPORT_PLAY_MEDIA
+        if self._source["type"] in ("Playlist", "Spotify"):
```

Comment:
Just curious why are you removing Cloud from the sources?

Reply:
The source `Cloud` doesn't exist on Linn devices and it should have been the `Spotify` source.

Reply:
Okay thanks for the clarification.

Reply:
It now enables people using the Spotify support on Linn devices to use skip and pause controls from Home Assistant which weren't there before.

Reply:
So not technically part of this PR...

Reply:
Well that can be added to the title and description of this PR.

Reply:
Just updated the title; thanks for updating the description! 

Sorry I totally forgot about that change - I spotted the bug during implementation and forgot.

Reply:
Thanks for that, no problem at all!

No worries about that ✌